python/erdos/operators/map.py

```diff
@@ -1,3 +1,5 @@
+import erdos
+

```

Comment:
add another newline (yapf should do this for you).

Reply:
ran yapf on all three files